14-include.md

```diff
@@ -0,0 +1,89 @@
+## Dependency on other compose projects
+
+A Compose project can declare dependency on another compose project, managed by another team
```

Comment:
I didn't find any reference to `Compose project` in the specification, to be consistent we should use `application` or `configuration`
```suggestion
A Compose application can declare dependency on another compose application, managed by another team
```